src/sentry/static/sentry/app/stores/hookStore.jsx

```diff
@@ -22,6 +22,8 @@ const validHookNames = new Set([
   'utils:onboarding-survey-url',
   'component:org-auth-view',
   'component:org-members-view',
+  'sidebar:bottom-items',
+  'sidebar:secondary-icon',
```

Comment:
Lets sort these up with the other sidebar items. I think each group here is sorted.